client/src/panels/decompositionPanel.js

```diff
@@ -361,7 +388,7 @@ class DecompositionPanel extends React.Component {
                 <MenuItem value="">
                   <em>None</em>
                 </MenuItem>
-                <MenuItem value="parameter" disabled={false}>
+                <MenuItem value="parameter" disabled={true}>
```

Comment:
Why are we no longer allowed to select parameters? That was a really nice feature.
@kylinmb

Reply:
Shoot! I wanted to talk to Ross about this before I committed it and forgot. This is another place where my insufficient understanding of MS and relying just on what I understand from the original study/paper is limiting me. The MS in previous work always used the dependent variable to build the regression lines. I was not sure if that was a requirement or just something they never did, and wanted to verify with Ross. If it was a requirement and part of the MS constraint then I am not sure it should be available. But again, I should have not added it to this pull request without verifying with those who would know.

Reply:
We've re-enabled M-S computation for design parameters which we want.